addons/pos_restaurant/static/src/js/multiprint.js

```diff
@@ -201,7 +195,7 @@ models.Order = models.Order.extend({
     },
     computeChanges: function(categories){
         var current_res = this.build_line_resume();
-        var old_res     = this.saved_resume || {};
+        var old_res     = this.saved_resume || [];
```

Comment:
This is fine, but we should be careful doing this. Both `current_res` and `old_res` are now Arrays instead of Objects. So as an example a little below this code there is a `for (line_hash in current_res)` loop. Doing this on Objects is fine, but on Arrays it is a bad practice [1]. So that should be fixed and the rest of the code should be looked at to see if there are no other issues that arise from changing these variables to be Objects.

[1] http://stackoverflow.com/questions/500504/why-is-using-for-in-with-array-iteration-such-a-bad-idea